assets/sass/homepage-style.scss

```diff
@@ -0,0 +1,4377 @@
+.gem-c-feedback__grid-row:after, fieldset:after, .form-section:after, .form-group:after, .notice:after, .clearfix:after, .metadata:after, .search:after, .js-accordion-with-descriptions .subsection-controls:after, .js-accordion-with-descriptions .subsection-wrapper:after, .gem-c-feedback__prompt:after {
```

Comment:
@jaroslavistok tieto skompilovane css sem prosim nedavaj lebo to preplacne vsetko ostatne CSSko co je tu ponastavovane.

Zmeny pridavaj do `assets/sass`